src/socket/tcp_server.cr

```diff
@@ -18,7 +18,9 @@ require "./tcp_socket"
 # ```
 #
 # Options:
-# - *backlog* to specify how many pending connections are allowed;
+# - *host* local interface to bind on, or `::` to bind on all local interfaces.
+# - *port* specific port to bind on, or `0` to receive an "ephemeral" (free, assigned by kernel) port.
```

Comment:
Does it work the same way on BSD/Darwin and on Windows?

Reply:
Yes. All supported operating systems' implementations are based on the BSD socket API.